_src/static/sass/utilities/_helpers.scss

```diff
@@ -104,9 +104,6 @@
 // reset of copy
 %reset-copy {
     font-family: $text-base-font-family;
-    font-size: font-size(base);
```

Comment:
NOTE: simplifying the base reset of copy since 1) we have copy styles now and 2) it was interfering with the copy styles' font-size cascade when it was extended from a ``<p>`` or similar atomic content piece of markup.